### ISSUE TITLE
'work as' dropdown on customize style page doesn't keep selection

### ISSUE THREAD

**Issue by highlander-ii** [dreamwidth/dreamwidth#3348]
from support request: https://www.dreamwidth.org/support/see_request?id=47440

steps to reproduce:

go to customize style -> switch to comm -> edit comm title -> save -> select 'edit' to change the comm title -> save -> open admin journal -> title on admin journal has changed

**Comment by incantatem1** [comment]
It's not just the title, it's literally everything

**Comment by alierak** [comment]
It appears this was the same issue as #3362 which was fixed by PR #3364 (not yet deployed in production).

**Comment by incantatem1** [comment]
Not yet deployed means it's not live on the site yet?

**Comment by momijizukamori** [comment]
Okay after some confused dev back-and-forth and testing, we've worked out the issue: #3364 is already deployed on the site but, importantly, only fixes the issue on https://www.dreamwidth.org/customize - NOT on https://www.dreamwidth.org/customize/options

So you should be able to edit journal titles and the page layout (along with picking a style) on the first link. I'm going to fix the issue for the second page, but it won't be live on the site until the next code push.

**Comment by incantatem1** [comment]
I'm not sure if that really fixes it. Earlier I tried creating a community/sideblog, and wanted to set a custom theme for it based on Complete Style. It did not even save the custom layer even though I followed all the familiar steps to install it, when accessing the "advanced" section it only showed the standard style (Neutral Good). Instead, my main blog had another broken wizard- (#number) in it. It has 4 of these which I guess are all from attempts to do this earlier while it was broken. 

The custom theme did briefly show up in the settings of the community, but when I tried to access it, I got redirected to my main blog and the custom theme was gone from the advanced section.

**Comment by momijizukamori** [comment]
Ah, I didn't realize you were having issues with custom layers - that's a different issue, which still needs some investigating.

**Comment by incantatem1** [comment]
It seems all to be related to the same core issue. Any settings done to communities not being applied to the right blog but instead redirected to the main.

**Comment by momijizukamori** [comment]
Yes, but the components that were responsible for the issues on /customize and /customize/options aren't on the /customize/advanced pages, so we need to dig to figure out what's expected/missing there.

**Comment by incantatem1** [comment]
I still think it's related. I'm able to choose styles in advanced for any community, but only ones I already made and saved. I was also able to install the Complete Style code. The problem only suddenly popped up when I wanted to select the custom layer in the "customize" settings. i saw the layer briefly, but when I wanted to access it for customizing, I got redirected to my main and the Complete Style broke and transformed into a "wizard-" thing. I don't think this directly has to do with the advanced section.
